lib/util.js

```diff
@@ -957,14 +958,18 @@ exports.log = function() {
 exports.inherits = function(ctor, superCtor) {
 
   if (ctor === undefined || ctor === null)
-    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'ctor', 'function');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'ctor', 'function',
+                               ctor);
 
   if (superCtor === undefined || superCtor === null)
-    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'superCtor', 'function');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'superCtor', 'function',
+                               superCtor);
 
   if (superCtor.prototype === undefined) {
-    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'superCtor.prototype',
-                               'function');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
+                               'superCtor.prototype',
+                               'function',
+                               superCtor.prototype);
```

Comment:
I personally prefer it this way to have a consistent way of doing this but I don't have strong feelings about it.